ipaserver/install/ipa_cacert_manage.py

```diff
@@ -216,7 +216,7 @@ def renew_external_step_2(self, ca, old_cert):
             options.external_cert_files, DN(old_cert.subject))
 
         with open(cert_file.name) as f:
```

Comment:
Thank you for catching this. A better and easier way to fix this is just to open the file for bytes-reading: `with open(cert_file.name, 'rb') as f:`